proto/cosmwasm/wasm/v1/tx.proto

```diff
@@ -72,6 +102,14 @@ message MsgInstantiateContractResponse {
   bytes data = 2;
 }
 
+// MsgInstantiateContract2Response return instantiation result data
+message MsgInstantiateContract2Response {
+  // Address is the bech32 address of the new contract instance.
+  string address = 1;
+  // Data contains base64-encoded bytes to returned from the contract
```

Comment:
"base64-encoded" is outdated, right? Sounds like copied from some JSON interface (same for `MsgInstantiateContractResponse` above).